test/beetle/message_test.rb

```diff
@@ -127,33 +128,42 @@ def setup
       end
     end
 
-    test "successful processing of a non redundant message should delete all keys from the database" do
+    test "successful processing of a non redundant message should delete all keys from the database (except the staus key, which should be set to expire)" do
       header = header_with_params({})
       header.expects(:ack)
       message = Message.new("somequeue", header, 'foo', :store => @store)
+      message.stubs(:simple?).returns(false)
 
       assert !message.expired?
       assert !message.redundant?
+      assert !message.simple?
```

Comment:
This assertion is just testing the stub above, which is unnecessary.

Reply:
fixed